src/01-functions.js

```diff
@@ -9,7 +9,16 @@
  *  alwaysTrue();
  *  //> true
  */
-function alwaysTrue() {}
+function alwaysTrue(a, b) {
```

Comment:
Whoa! Looks like we're doing more here that the problem called for. There were no parameters required here(`a,b`)  and we simply needed to `return` the boolean `true` here.